guides/accounting-guide.rst

```diff
@@ -61,7 +61,27 @@ Installing Software Packages
 ============================
 
 The ``flux-accounting`` package should be installed on the management node
-from your Linux distribution package manager.
+from your Linux distribution package manager. Once installed, the service
+that accepts ``flux account`` commands and interacts with the flux-accounting
+database can be started.
+
+You can enable the service with ``systemctl``; if not configured with a custom
+path, the flux-accounting systemd unit file will be installed to the same
+location as flux-core's systemd unit file:
+
+.. code-block:: console
+
+  $ sudo systemctl enable /path/to/flux-accounting.service
```

Comment:
The path should not be necessary here if the unit file is installed in the right place, which will be the case when installing from packages.   `sudo systemctl enable flux-accounting` should be sufficient.